static/js/resize.js

```diff
@@ -23,7 +23,7 @@ function size_blocks(blocks, usable_height) {
     _.each(blocks, function (block) {
         var ratio = block.real_height / sum_height;
         ratio = confine_to_range(0.05, ratio, 0.85);
```

Comment:
Also, should I make a new PR with the detailed commit message? Or is it possible to edit my commit message somehow? I searched online to realize that in order to edit commit messages, I'll have to use <code>git rebase</code>.